### DISCUSSION TITLE
Help getting started as a contributor.

### DISCUSSION THREAD

**Discussion by bswhite1** [firebase/flutterfire#5725] [Q&A]
I am trying to add some debugging to diagnose some odd behavior we are seeing in our app that uses flutterfire. I am having extreme difficulty however just getting this to run and build after pulling the repo.

Are there any more detailed help guides than the current Contributing section? 

I am using Android Studio IDE. Is there a list of settings to configure that? I have errors even in my build.gradle "Cannot resolve symbol 'GradleException'"
It also concerns me that the melos command, the build.yaml had errors. The path for windows was incorrect. Was that designed for windows use?

Any help would be greatly appreciated.

**Comment by Jaydyzzle813** [comment]
I’m sorry but I just deleted all the apps I had or I would’ve been more
than happy to help
